modules/end-user-guide/pages/using-a-git-provider-access-token.adoc

```diff
@@ -25,6 +25,11 @@ pass:[<!-- vale RedHat.Spelling = YES -->]
 
 You can create and apply multiple access-token Secrets per Git provider. You must apply each of those Secrets in your user {orch-namespace}.
 
+[NOTE]
+====
+You can configure personal access tokens in *User Preferences* page (`pass:c,a,q[{prod-url}]/dashboard/#/user-preferences?tab=personal-access-tokens`) of your {prod-short} dashboard.
+====
+
 .Prerequisites
 
 * You have cluster administrator permissions for the cluster on which your organization's {prod-short} instance is running.
```

Comment:
@vinokurig wondering why is it required to have cluster-admin permissions?

Reply:
That's probably a leftover, @akurinnoy please remove the item.